lighthouse-core/audits/consistently-interactive.js

```diff
@@ -247,8 +246,7 @@ class ConsistentlyInteractiveMetric extends Audit {
           displayValue: Util.formatMilliseconds(timeInMs),
           optimalValue: this.meta.optimalValue,
           extendedInfo: {
-            value: extendedInfo,
-            formatter: Formatter.SUPPORTED_FORMATS.NULL,
+            value: extendedInfo
```

Comment:
> why you gotta nuke the extra commas? :(

haha, I realized what I was doing halfway through and thought that might be a mistake. I'll restore them :)